schemas/system/Log.schema

```diff
@@ -1,7 +1,6 @@
 Log({
   User: { category: 'User', required: true },
-  Category: { domain: 'Id', required: true },
+  Identifier: { category: 'Identifier', required: true },
   Timestamp: { domain: 'Timestamp', required: true },
-  Operation: { domain: 'UserAccess', required: true },
-  Ids: { domain: 'List' }
+  Operation: { domain: 'UserAccess', required: true }
```

Comment:
`Log.Identifier` will point to touched entity

Reply:
@tshemsedinov but is there a way to determine what data was updated or inserted as of the moment of insertion/update?
Because now using `Log.Identifier` you can get the latest version, but it may not be the one inserted/updated corresponding to this `Log` record.

Reply:
Understood, I'll add JSON field to store changes